src/jobsautoreport/slack.py

```diff
@@ -13,30 +15,30 @@ def __init__(self, web_client: WebClient, channel_id: str) -> None:
 
     def send_report(self, data: dict[str, Any]) -> None:
         if not self._is_valid(data):
+            logger.debug("At least one value is missing")
             return
 
-        blocks_list = self._format_as_message(data)
-        res = self._client.chat_postMessage(
-            channel=self._channel_id, blocks=blocks_list
-        )
-        time_stamp = res["ts"]
+        blocks = self._format_msg(data)
+        logger.debug("message format finished successfully")
+        res = self._client.chat_postMessage(channel=self._channel_id, blocks=blocks)
+        logger.info("msg sent successfully")
 
-    def comment_on_message(
-        self, msg_time_stamp: str, data: list[dict[str, Any]]
-    ) -> None:
-        blocks_list = self._format_as_comment(data)
+        time_stamp = res["ts"]
+        blocks = self._format_comment(data)
+        logger.debug("comment format finished successfully")
         res = self._client.chat_postMessage(
-            channel=self._channel_id, blocks=blocks_list, ts=msg_time_stamp
+            channel=self._channel_id, blocks=blocks, thread_ts=time_stamp
         )
+        logger.info("comment sent successfully")
 
-    @staticmethod
-    def _format_as_message(data: dict[str, Any]) -> list[dict[str, Any]]:
+    @classmethod
+    def _format_msg(cls, data: dict[str, Any]) -> list[dict[str, Any]]:
```

Comment:
```suggestion
    def _format_message(cls, data: dict[str, Any]) -> list[dict[str, Any]]:
```
I suggest you to avoid using abbreviations as much as possible, it makes the code less readable